RELEASENOTE.md

```diff
@@ -1,8 +1,14 @@
-Version Notes (Current Version: v2.46)
+Version Notes (Current Version: v2.45.1)
 =======================================
 
-v2.46(Pre-Release)
+v2.45.1
 -----------
+1. Improve performance by upgrading Jint to 2.11.58. (#5032)
+1. Bug fix:
+    1. Fix incremental build error when previous build has no content. (#5010)
```

Comment:
stable xref resolve result?

Reply:
I am thinking of excluding this change due to code freeze...

Reply:
added